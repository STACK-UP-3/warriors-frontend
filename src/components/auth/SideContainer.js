import React from 'react';

export default (props) => (
  <div className='sideContainer'>
    <div className='background'></div>

    <div className='content'>
      <h3>{props.title}</h3>
      {props.subtitle}
    </div>
  </div>
);
