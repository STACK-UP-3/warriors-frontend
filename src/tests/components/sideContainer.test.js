import React from 'react';
import { shallow } from 'enzyme';
import SideContainer from '../../components/auth/SideContainer';

describe('*************** Testing the SideContainer  component ***************', () => {
  it('Should render the SideContainer page correctly', () => {
    const wrapper = shallow(
      <SideContainer title='Welcome to BR' subtitle='signup here' />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
