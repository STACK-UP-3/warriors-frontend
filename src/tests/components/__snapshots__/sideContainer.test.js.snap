// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`*************** Testing the SideContainer  component *************** Should render the SideContainer page correctly 1`] = `
<div
  className="sideContainer"
>
  <div
    className="background"
    style={
      Object {
        "backgroundImage": "url([object Object])",
      }
    }
  />
  <div
    className="content"
  >
    <h2>
      Welcome to BR
    </h2>
    signup here
  </div>
</div>
`;
